docs/source/examples/pec-tutorial.myst

```diff
@@ -10,7 +10,7 @@ kernelspec:
   language: python
   name: python3
 ---
-(label-pec-example)=
+
```

Comment:
Fine to delete it, to avoid overlapping content with the main docs. In the future we can add a new PEC tutorial that highlights something different with respect to the main docs.